docs/resources/aws_cloudformation_stacks.md

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
this doc link needs to be added to https://github.com/inspec/inspec-aws#resource-documentation

Reply:
Is this for me to do? in the unlikely case that it is, how is it done? Did not see the process before

Reply:
Yes, it is a manual process, please edit it in the [README.md](https://github.com/inspec/inspec-aws/blob/master/README.md)
Also, could you please add the docs link for `ecrpublic` (your previous PR) .. I might have missed it than..